### ISSUE TITLE
fetching list of repos is unreliable

### ISSUE THREAD

**Issue by codebox** [gjtorikian/addalicense.com#34]
Retrieving list of repos from GitHub is unreliable - frequently hangs and need to refresh the page, sometimes it works and sometimes not

**Comment by MaxPleaner** [comment]
Yes, I have tried to this twice and nothing happens. I wait a while, like 15 minutes.

**Comment by sman591** [comment]
Same thing happens to me. Looking at the web console, I think this is due to #32
